pyshelf/metadata/cloud_portal.py

```diff
@@ -0,0 +1,39 @@
+import copy
+
+
+class CloudPortal(object):
+    def __init__(self, container):
+        """
+            Args:
+                container(pyshelf.metadata.container.Container)
+        """
+        self.codec = container.yaml_codec
+        self.mapper = container.mapper
+        self.container = container
+
+    def update(self, cloud_identifier, metadata):
```

Comment:
Just a note for the entirety of this pull req. Do we want all public functions to have doc blocks? If so there a few in that don't have them. I like it even if it is obvious what the function does. I know I forget to all the time but I personally like it and think we should do it as much as possible. Whatever you think though.